### PR TITLE
Adjust testing bundle.

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- adjust testing bundle

**Related issue(s)**

### CODE REVIEW

File: bundles/testing-0.0.1/README.md [hunk 1]
```diff
@@ -7,5 +7,7 @@ The `Testing` bundle is a simple bundle which contains a ConfigMap and a Deploym
 ## Details
 
 The bundle contains the following plans:
-- Minimal, which contains only necessary parameters. There is no binding template defined so binding does not work for this plan.
+- Minimal, a not bindable plan which contains only necessary parameters.
 - Full, which contains all possible parameters.
+
+The binding contains one value with the ConfigMap name.
```
Comment (klaudiagrz) [inline bundles/testing-0.0.1/README.md]:
```suggestion
Binding returns one value with the ConfigMap name.
```

File: bundles/testing-0.0.1/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 name: testing
 version: 0.0.1
 id: faebbe18-0a84-11e9-ab14-d663bd873d94
-description: "Testing bundle which contains dummy ConfigMap"
+description: "A simple bundle which contains a ConfigMap and a Deployment. The binding contains one value with the ConfigMap name."
```
Comment (klaudiagrz) [inline bundles/testing-0.0.1/meta.yaml]:
```suggestion
description: "A simple bundle which contains a ConfigMap and a Deployment. Binding returns one value with the ConfigMap name."
```

File: bundles/testing-0.0.1/meta.yaml [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 displayName: "Testing bundle"
 
 tags: test
 labels:
   showcase: "true"
 providerDisplayName: SAP
-longDescription: "Testing bundle which contains dummy ConfigMap"
-bindable: true
+longDescription: "A simple bundle which contains a ConfigMap and a Deployment. The binding contains one value with the ConfigMap name."
```
Comment (klaudiagrz) [inline bundles/testing-0.0.1/meta.yaml]:
```suggestion
longDescription: "A simple bundle which contains a ConfigMap and a Deployment. Binding returns one value with the ConfigMap name."
```

File: bundles/testing-0.0.1/plans/full/bind-instance-schema.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
   "properties": {
     "exampleField": {
       "type": "string",
-      "title": "Additional data"
+      "title": "Additional data",
+      "description": "Example attribute which does nothing."
```
Comment (klaudiagrz) [inline bundles/testing-0.0.1/plans/full/bind-instance-schema.json]:
```suggestion
      "description": "Example attribute which has no function."
```

File: bundles/testing-0.0.1/plans/full/create-instance-schema.json [hunk 5]
```diff
@@ -4,11 +4,13 @@
   "properties": {
     "additionalData": {
       "type": "string",
-      "title": "Additional data"
+      "title": "Additional data",
+      "description": "Example attribute which does nothing."
```
Comment (klaudiagrz) [inline bundles/testing-0.0.1/plans/full/create-instance-schema.json]:
```suggestion
      "description": "Example attribute which has no function."
```

File: bundles/testing-0.0.1/plans/full/update-instance-schema.json [hunk 6]
```diff
@@ -4,7 +4,8 @@
   "properties": {
     "additionalData": {
       "type": "string",
-      "title": "Additional data"
+      "title": "Additional data",
+      "description": "Example attribute which does nothing."
```
Comment (klaudiagrz) [inline bundles/testing-0.0.1/plans/full/update-instance-schema.json]:
```suggestion
      "description": "Example attribute which has no function."
```

File: scripts/generate_changelog.sh [hunk 7]
```diff
@@ -27,6 +27,3 @@ docker run --rm -v ${BUNDLES_PATH}:/repository -w /repository -e NEW_RELEASE_TIT
```
Comment (jasiu001) [inline scripts/generate_changelog.sh]: I also propose remove line 25-26 `# generate full changelog`. If we don't push changelog.md file to repository anymore then we don't have to generate it. Also we can remove file `push_changelog.sh` if we don't use it anymore.

**Pull request merged**